spring-cloud-gcp-pubsub/pom.xml

```diff
@@ -30,6 +30,10 @@
 			<artifactId>spring-integration-core</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
```

Comment:
I guess `optional` as well?

Reply:
@meltsufin what would you prefer? make it non-optional again?

Reply:
optional is fine, we just need to document that the dependency is needed if you want to take advantage of object to json conversion.

Reply:
done